### PR TITLE
opened issue #2201(Added prim algorithm)

### LINKED ISSUE #2201: Prims Algorithm
- [x] C
- [x] C++
- [ ] C# - @ritish099
- [ ] CoffeeScript
- [ ] Dart - @Swarnimashukla
- [ ] Go - @adi44
- [x] Java
- [x] JS - @sukritishah15
- [x] Kotlin - @darkmatter18
- [ ] Php - @prince-09
- [x] Python
- [x] Readme
- [ ] Ruby
- [ ] Rust
- [ ] TypeScript - @BhuwanChandra

### PR DESCRIPTION
**Pull request by aryangulati**
Fixes #2201

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [x] Input is taken dynamically.
- [x] Sample Input / Output is added at the end of file.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

#### Changes proposed in this pull request:
- added prim algorithm in JavaScript with I/O
- added prim algorithm in c# with input &output
 
#### Languages Used:
- JavaScript (js) 
- C#(cs) 

#### Files Added:
- Prims_Algorithm/Prims_Algorithm.js
-  Prims_Algorithm/Prims_Algorithm.cs

> We're happy to help you get this ready -- don't be afraid to ask for help.

Thanks!

### CODE REVIEW

**Comment by raghav-dalmia** [comment]
complete your PR template.

**Comment by aryangulati** [comment]
PR TEMPLATE MEANS??

**Comment by aryangulati** [comment]
Is I have to close request??

**Comment by raghav-dalmia** [comment]
complete your pr template. add issue number. squash your commits.

**Comment by aryangulati** [comment]
Done

File: Prims_Algorithm/Prims_Algorithm.cs [hunk 1]
```diff
@@ -0,0 +1,80 @@
+private static int MinKey(int[] key, bool[] set, int verticesCount)
+{
+	int min = int.MaxValue, minIndex = 0;
```
Comment (jainaman224) [inline Prims_Algorithm/Prims_Algorithm.cs]: Incorrect indentation. Use 4 space indentation.

File: Prims_Algorithm/Prims_Algorithm.cs [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+private static void Print(int[] parent, int[,] graph, int verticesCount)
+{
+	Console.WriteLine("Edge     Weight");
+	for (int i = 1; i < verticesCount; ++i)
+		Console.WriteLine("{0} - {1}    {2}", parent[i], i, graph[i, parent[i]]);
+}
+
+public static void Prim(int[,] graph, int verticesCount)
+{
+	int[] parent = new int[verticesCount];
```
Comment (jainaman224) [inline Prims_Algorithm/Prims_Algorithm.cs]: There is no data present. How to run this program.

**Comment by jainaman224** [comment]
@aryangulati Are you working on it?

**Comment by somya-kapoor** [comment]
@aryangulati Are you working on it?

**Comment by somya-kapoor** [comment]
@aryangulati Due to no activity, I am closing this issue.

**Pull request closed without merging**